index.html

```diff
@@ -182,6 +182,21 @@ <h3 id="unacceptablebehaviour">
         <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
       </ul>
 
+      <p>The enforcers of this Code of Conduct should prioritise marginalized people’s safety over privileged people’s comfort,
```

Comment:
> Code of Conduct

This is not formally called that, suggest reducing to "Code".

marginalized people vs privileged people - can we find a better expression?

1. They're not necessarily mutually exclusive groups
2. The use of the plural suggests that it is about groups of people not individuals

I think this is about safety and marginalization primarily, so perhaps something like this would be better?

> The enforcers of this Code should prioritise the safety of individuals within marginalised communities over the comfort of others,

Reply:
Thank you 👍

Reply:
Resolved in 8cd6287